packages/zoltan2/src/algorithms/order/Zoltan2_AlgAMD.hpp

```diff
@@ -146,6 +149,10 @@ class AlgAMD : public Algorithm<Adapter>
       // wgts are ignored in AMD
       model->getEdgeList(edgeIds, offsets, wgts);
 
+#if AMD_WITH_LNO_T
+    {
+
+      // SR: We will always call AMD with lno_t
```

Comment:
From the comments, it sounds as is SuiteSparse can be compiled with SuiteSparse_long == long or SuiteSparse_long == long long.  Is that correct?
If so, what happens in this case when lno_t == long long but SuiteSparse_long = long?  Will the AMD_WITH_LNO_T version work?